.pax/prepare-workspace.sh

```diff
@@ -23,6 +23,7 @@ set -x
 
 # expected workspace layout:
```

Comment:
had to think again what and where this was. Can you update this line to say the following and remove all doubt
`# expected input workspace layout ($ROOT_DIR):`

Reply:
Done.